docs/security.md

```diff
@@ -154,8 +149,15 @@ authorization:
   ldapConfigPath: '<ldap_config_path>'
 ```
 
-The LDAP config file should have the following contents:
+**Caution:**
+The regex pattern matching for roles only works for API calls, and not on the webapp. 
+The webapp only supports ADMIN, USER, and API roles. 
+For instance, in the case that a user has `privileges: SUPER_USER_API` and the authorization section includes `admin: (.*)SUPER(.*)` the user will *not* have ADMIN role privileges in the webapp. 
```

Comment:
What I understood is the the result of `lbLdapClient.getMemberOf()` will override the privilege setting in presetUsers.
https://github.com/trinodb/trino-gateway/blob/acca2ea637fd673af1bef84fe27a0b482ea1ada0/gateway-ha/src/main/java/io/trino/gateway/ha/security/AuthorizationManager.java#L46-L51